source/extensions/quic_listeners/quiche/platform/quic_flags_impl.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy
+
+#include <string>
+#include <vector>
+
+// This is a dummy implementation which just allows its dependency to build.
+// TODO(mpwarres): implement once QUICHE flag mechanism is defined.
+
+extern bool FLAGS_quic_supports_tls_handshake;
+#define GetQuicFlagImpl(flag) ({ false; })
+#define SetQuicFlagImpl(flag, value)                                                               \
```

Comment:
If this is going to be eventually replaced with a non-macro construct then it's worth a comment saying so.

If this is always going to be a macro, can we use MACRO_SYNTAX per Envoy style guide?

Reply:
This will be implemented in on-going PR:https://github.com/envoyproxy/envoy/pull/6247. It currently seems to be macro but I think it's possible to be changed into non-macro construct.

Reply:
OK leave TODO then to replace with non-macro.